inc/namespace.php

```diff
@@ -50,3 +52,62 @@ function load_publication_checklist() {
 
 	require_once Altis\ROOT_DIR . '/vendor/humanmade/publication-checklist/plugin.php';
 }
+
+/**
+ * Load Yoast Duplicate Posts, if enabled.
+ */
+function load_duplicate_posts() {
+	$config = Altis\get_config()['modules']['workflow']['clone-republish'] ?? null;
+
+	// Bail if Clone & Republish is disabled.
+	if ( ! $config ) {
+		return;
+	}
+
+	// Load the main plugin file.
+	require_once Altis\ROOT_DIR . '/vendor/yoast/duplicate-post/duplicate-post.php';
+
+	// Load all the other plugin files. This is normally handled internally by a Composer autoloader.
+	require_once Altis\ROOT_DIR . '/vendor/yoast/duplicate-post/src/admin/options.php';
```

Comment:
Thought I'd check this after what you mentioned on standup. This won't be maintainable and I think might have been an issue with the composer autoloader files not regenerating properly when you added the plugin. You should be able to remove all these, and try running `composer dump-autoload`, if that doesn't work try removing the whole vendor directory then doing a `composer install`.

Composer should definitely handle this, it can just be a bit confusing working locally sometimes when you add a new dependency and the autoloaders don't update.

Reply:
Cool, that worked. That was a fun rabbit hole. 🙄